src/string.js

```diff
@@ -117,12 +117,15 @@ String.prototype.alternatingCase = function () {
 String.prototype.getMiddle = function () {
     //if string is just one character return that character 
     if (this.length === 1) return this[0];
+
     //split string into two
     const regex = new RegExp(`.{${ ~~(this.length / 2) }}`,"g"),
         matches = this.match(regex);
```

Comment:
### Nitpick

Each declaration should be on it's own line, refer tho the airbnb spec for clarification.